test/requirements.txt

```diff
@@ -0,0 +1,6 @@
+python3 -m venv gym-supermariobros-env
+source gym-supermariobros-env/bin/activate
+
+# Installation Reqs
+pip install gym-super-mario-bros
+pip install gym pyvirtualdisplay
```

Comment:
You should avoid installing individual packages in your setup.sh file. Instead your requirements.txt file should include
```
gym-super-mario-bros
gym
pyvirtualdisplay
```